subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -335,6 +335,15 @@ config BT_CTLR_ADV_AUX_SET
 	help
 	  Maximum supported advertising auxiliary channel sets.
 
+config BT_CTLR_ADV_USE_MAX_SECONDARY_INTERVAL
+	bool "Use the maximum possible interval between ADV_AUX_INDs"
+	depends on BT_BROADCASTER && BT_CTLR_ADV_EXT
+	default n
+	help
+	  Use the value of Secondary_Advertising_Max_Skip to only send
+	  ADV_AUX_IND when required - ie. skip as many as allowed by the
+	  parameters set by the host.
+
```

Comment:
If this is not something configurable in IXIT or ICS, lets not have a Kconfig (in this PR Host will always provide a valid value anyway).

Reply:
True, that would be preferable; However, this has some implications for qualification (specifically LL/DDI/ADV/BV-28-C). I'll see if I can get some time to look at it next week, but otherwise it will have to wait until I get back from my summer vacation (ie. late August).

Reply:
Changed the implementation to avoid the configuration flag; This means using a slightly different calculation for max_skip == 0 to avoid a change in behaviour while still passing LL/DDI/ADV/BV-28-C for higher values of max_skip